rtl/include/eclhelper.hpp

```diff
@@ -2337,7 +2339,8 @@ struct IHThorIndexReadBaseArg : extends IHThorCompoundBaseArg
     virtual IOutputMetaData * queryDiskRecordSize() = 0;            // Expected layout
     virtual IOutputMetaData * queryProjectedDiskRecordSize() = 0;   // Projected layout
     virtual unsigned getFlags() = 0;
-    virtual unsigned getFormatCrc() = 0;
+    virtual unsigned getFormatCrc() = 0;                             // Corresponding to projectedDiskRecordSize, and should be renamed
```

Comment:
This should really be renamed at the same time, so you can check that the correct function is being called - by roxie and thor.